modules/button/css/README.md

```diff
@@ -291,6 +292,31 @@ Apply primary styling to the button by adding `.wdc-btn-primary` to the buttons
 <button class="wdc-btn-text wdc-btn-text-inverse">Hovered Button</button>
 ```
 
+### Inline Icon
+You can use a [System Icon](/?path=/story/css-icon--system-icon) either before or after the text of Primary, Secondary, Delete and Text buttons. The class `.wdc-icon-position-left` (default) or  `.wdc-icon-position-right` adjust the margin of the icon.
+
+`System Icon` requires a JavaScript import for the injector:
+
+```javascript
+import initializeIcons from '@workday/canvas-kit-css-icon';
+
+initializeIcons();
+```
+
+```html
+<button className="wdc-btn wdc-btn-primary wdc-btn-size-l">
+  <i className="wdc-icon" data-icon="edit" data-category="system" />Edit
+</button>
+
+
+<a href="#" className="wdc-btn-text">
+  <i className="wdc-icon wdc-icon-position-left" data-icon="edit" data-category="system" /> Edit
```

Comment:
Can we drop this example? It's essentially the same as the first. I think we only need the first and third